presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1185,6 +1186,11 @@ public SystemSessionProperties(
                         "When enabled and reading from materialized view, partition stitching is applied to achieve data consistency",
                         featuresConfig.isMaterializedViewDataConsistencyEnabled(),
                         false),
+                booleanProperty(
```

Comment:
I think we should just enable it by default. @jainxrohit what do you think?

Reply:
Yes enabled by default. We need this config so that we can disable it if this runs in to any issues.

Reply:
Is it not enabled by default as written? `featuresConfig.isMarerializedViewDataConsistencyEnabled()` resolves to `true` from what I can tell.